notes/roman-urdu/79_Surah_Nazi'at.md

```diff
@@ -0,0 +1,38 @@
+# Number 79: Surah Nazi'at
+
+Classification: Makki  
+Rukus: 2  
+Ayahs: 46  
+Subject: Resurrection  
+Related Surah: Surah Abas  
+
```

Comment:
Is there a background to this surah?

Reply:
I have added background based on Dr. Israr's video. We can add background to any Surah if we know add study them better. For now, I would say no. Does this makes sense?